interface/github.py

```diff
@@ -45,6 +45,54 @@ def org_has_member(self, username):
         except GithubException as e:
             raise GithubAPIException(e.data)
 
+# ---------------------------------------------------------------
+# --------------- methods related to team members ---------------
+# ---------------------------------------------------------------
+
+    def list_team_members(self, team_id):
+        """Return a list of users in the team of id team_id."""
+        try:
+            team = self.github.get_team(team_id)
+            # Question: Should we return
+            # github.PaginatedList.PaginatedList or just
+            # regular python's list.
```

Comment:
Actually now that I think about it, I think it seems like it should be okay to return `PaginatedList` since it has the same interface as normal python's list and you don't have to import anything to be able to use it. Let's talk about it when ur here.